site/content/docs/user/known-issues.md

```diff
@@ -207,7 +207,7 @@ Dec 07 00:37:53 kind-1-control-plane kubelet[688]: I1207 00:37:53.229561     688
 Dec 07 00:37:53 kind-1-control-plane kubelet[688]: E1207 00:37:53.229638     688 eviction_manager.go:351] eviction manager: eviction thresholds have been met, but no pods are active to evict
 ```
 ## Failure to Create Cluster with Cgroups v2
-Support for Cgroups v2 was introduced in Kubernetes 1.19 (see [release notes](https://v1-19.docs.kubernetes.io/docs/setup/release/notes/)). Accordingly, only Kubernetes versions >= 1.19 are supported on hosts using Cgroups v2.
+Support for Cgroups v2 was introduced in Kubernetes 1.19 (see [release notes](https://v1-19.docs.kubernetes.io/docs/setup/release/notes/)). Docker Desktop 4.3.0+ uses cgroupsv2 and can run only Kubernetes versions >= v1.19. Accordingly, only Kubernetes versions >= 1.19 are supported on hosts using Cgroups v2.
```

Comment:
The sentence is redundant, if docker desktop uses cgroupsv2 then can only run in kubernetes versions > v1.19

However, if users already know they are using cgroupsv2 and they check this subsection :shrug:

Reply:
Could you please take a look on this comment https://github.com/kubernetes-sigs/kind/issues/2629#issuecomment-1048327920

Reply:
yeah, the point I'm trying to made is that the known issue should be a section `Docker Desktop 4.3.0+` and inside somethink like " docker desktop uses cgroupsv2 ...  , link to the release notes of docker desktop ... and  only works with k>1.19 ... ink to subsection "Failure to create cluster with cgroups"

Reply:
can you add it as a subsection instead?